src/notation/internal/excerptnotation.cpp

```diff
@@ -94,10 +94,18 @@ void ExcerptNotation::fillWithDefaultInfo()
         }
     };
 
-    setText(TextStyleType::TITLE, qtrc("notation", "Title"));
-    setText(TextStyleType::COMPOSER, qtrc("notation", "Composer / arranger"));
-    setText(TextStyleType::SUBTITLE, "");
-    setText(TextStyleType::POET, "");
+    auto getText = [&score](TextStyleType textType, const QString& defaultText) {
+        if (Text* t = score->getText(textType)) {
+            return t->plainText().toQString();
+        } else {
+            return defaultText;
+        }
+    };
+
+    setText(TextStyleType::TITLE, qtrc("notation", getText(TextStyleType::TITLE, "")));
+    setText(TextStyleType::COMPOSER, qtrc("notation", getText(TextStyleType::COMPOSER, "")));
+    setText(TextStyleType::SUBTITLE, qtrc("notation", getText(TextStyleType::SUBTITLE, "")));
+    setText(TextStyleType::POET, qtrc("notation", getText(TextStyleType::POET, "")));
```

Comment:
I don't think `qtrc` should be here. The values of `getText(…)` are potentially not translatable strings; some of them might be even entered by the user. So we should not try to translate them, otherwise we loose performance and might get strange results (for example, if the user enters something that _does_ happen to be recognised by the translation system).

Reply:
Ah, good point, will fix that.